bundle/language_en.properties

```
@@ -829,10 +829,13 @@ angal.medicalstock.multiplecharging.chargetype
 angal.medicalstock.multiplecharging.chooseamedical                                                     = Choose a Medical
 angal.medicalstock.multiplecharging.cost                                                               = Cost
 angal.medicalstock.multiplecharging.datecannotbebeforelastmovementdate.msg                             = Date cannot be before last movement date.
+angal.medicalstock.multiplecharging.empty                                                              = Empty
```

```suggestion
angal.medicalstock.multiplecharging.empty.txt                                                          = Empty
```
Technically all message constants should have some kind of suffix: txt, btn, key, msg, fmt.msg, etc

See `expired` and `hideemptylots` below for other examples.

---

I realize now the all the other bundles around have no suffix... but in this case what suffix we should apply?

---

not sure about the suffix to use, we will do a revision in another issue.